src/data/data.js

```diff
@@ -0,0 +1,97 @@
+const City = require('../models/City');
+const Country = require('../models/Country');
+const Actor = require('../models/Actor');
+const Genre = require('../models/Genre');
+
+let data = [];
+
+const belarus = new Country({
```

Comment:
you should have the list of objects 

for **country** 
```
{
  name: "Belarus",
}
```

for **city**
you should make up how you will   find the ids of countries
```
{
  name: "Lida"
  countryName: 
}
```
and so on

after that you should go through the arrays of those objects with `.forEach` and create documents.

it's better to have separated arrays for every collection, because you should handle them in a different way